packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts

```diff
@@ -26,7 +26,7 @@ import { TextAlignPlugin } from '../../slate/TextAlignPlugin'
 import { BlockPlugin } from '../../slate/BlockPlugin'
 import { TypographyPlugin } from '../../slate/TypographyPlugin'
 import { InlineModePlugin } from '../../slate/InlineModePlugin'
-import { toText } from '../../slate'
+import { toText } from '../../slate/utils'
```

Comment:
Importing `slate/index` was the culprit here.

Reply:
Dang, this is such an easy mistake to make. I wonder if we can automate an eslint or another check to make sure we don't accidentally introduce something like this in the future again.